src/main/deparse.c

```diff
@@ -501,11 +501,6 @@ SEXP attribute_hidden do_dump(SEXP call, SEXP op, SEXP args, SEXP rho)
 
 static void linebreak(Rboolean *lbreak, LocalParseData *d)
 {
-#ifdef ENABLE_DYNTRACE
```

Comment:
Could you explain this?

Reply:
The last line of this function, `writeline`, appends a newline to the buffer which contains the part of the expression serialized so far. `ifdef` code bails out immediately from this function preventing the linebreaks from getting added.